xwiki-platform-core/xwiki-platform-web/src/main/webapp/resources/js/xwiki/editors/dataeditors.css

```diff
@@ -323,7 +323,10 @@ div#xwikiobjects input[type="radio"], div#xwikiclassproperties input[type="radio
   text-decoration: underline;
   background-color: $theme.highlightColor;
 }
-
+#add_xobject .buttonwrapper, #add_xproperty .buttonwrapper{
+  position: relative;
+  top: -2px;
```

Comment:
This feels like a hack. Why ``-2`` and not ``-3``. Did you test on all [supported browsers](https://dev.xwiki.org/xwiki/bin/view/Community/SupportStrategy/BrowserSupportStrategy)? It needs at least a comment explaining how you ended up with this value (and the reason can't be "because it works"). You couldn't fix this with ``vertical-align``?

Reply:
Yes, that can also be fixed using` vertical-align: top`  for the 'Add' button. If this seems to be a better solution I will make the changes.

Reply:
Using ``vertical-align`` is better than using magic numbers.

Reply:
I have tested them on Chrome, Edge, and Firefox and it workes perfectly. 
**Result:** 
![Result](https://user-images.githubusercontent.com/40354433/76096232-8fa10500-5fe7-11ea-9c3b-d4375e41a889.PNG)

**Inspect:** 

![Edge-inspect](https://user-images.githubusercontent.com/40354433/76096285-a47d9880-5fe7-11ea-81bf-ef4ac069e190.PNG)